introduction.md

```diff
@@ -0,0 +1 @@
+This is a simple intoduction and I just want to say hi no more no less.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.